### PR TITLE
Fixes bug where you cant change launch url

### PR DESCRIPTION
**Pull request by vertein**

### CODE REVIEW

**Review by ChristianMurphy — APPROVED** [review]
Not entirely sure what was broken.
still LGTM

**Comment by vertein** [comment]
@ChristianMurphy, if you check out our [test](https://test.my.wisc.edu/widget-creator) instance of widget-creator and create a 'list of links' widget, I noticed that the launch url for the bottom of the widget isn't customize-able, so it always goes to example.com.  I figured it might be a little nicer if the widget creator was a little more robust

**Review by ChristianMurphy — APPROVED** [review]
Looks even better now I know whats being fixed :stars:

File: src/main/webapp/my-app/my-app.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	padding-bottom: 20px;
 }
 
-.widget-creator widget {
+.widget-creator preview-widget {
```
Comment (davidmsibley) [inline src/main/webapp/my-app/my-app.css:9]: whoops, good catch

**Review by davidmsibley — APPROVED** [review]

**Pull request merged**